sdv/tabular/base.py

```diff
@@ -390,6 +391,19 @@ def _conditionally_sample_rows(self, dataframe, max_retries, max_rows_multiplier
 
         return sampled_rows
 
+    def _randomize_samples(self, randomize_samples):
```

Comment:
I don't see this used anywhere apart from the unit tests.
On the other hand, what happens if `randomize_samples` is `True`?
Should we set the seed to None?

Reply:
@csala Yes, if `randomize_samples` is true, we should set the seed to `None`. I updated the code to call the `_randomize_samples` method, and I also added an integration test to test that setting the seed to None works as expected.